### ISSUE TITLE
Zonder JS: rare interface

### ISSUE THREAD

**Issue by krijnhoetmer** [EmielZuurbier/web-audio#8]
Hou er rekening mee dat de interface niet heel veel toevoegt zonder JavaScript, of als je JavaScript breekt. Misschien alle controls ook juist toevoegen als je browser wel de geavanceerde opties ondersteunt?

**Comment by EmielZuurbier** [comment]
Yes, good point! Ik heb er nu een `<noscript>` tag ingevoegd en de form standaard uitgezet. Wanneer de API wordt ondersteund komt deze tevoorschijn.

**Comment by EmielZuurbier** [comment]
PS: 

> rare interface ??

**Comment by krijnhoetmer** [comment]
Raar als in verwarrend voor de bezoeker; een interface die niets doet is raar :)

`<noscript>` is niet de netste oplossing natuurlijk, maar op deze manier, met het verborgen formulier, is het in ieder geval minder verwarrend.

Probeer `<noscript>` in de toekomst wel te vergelijken met de periode toen de iPhone net uitkwam en Flash niet ondersteund werd. Sommige bezoekers hebben niet 'de keuze' om iets te upgraden of 'aan' te zetten. `<noscript>` had daarom misschien beter `<ikgeeftoedatikietsverkeerdoe>` kunnen heten :)
